jupyterlab/commands.py

```diff
@@ -2230,10 +2230,13 @@ def noop(x, y, z):
     return return_value
 
 
-def _is_disabled(name, disabled=[]):
+def _is_disabled(name, disabled={}):
```

Comment:
Need to double check whether a `list` or a `dict` is expected here, based on the change in https://github.com/jupyterlab/jupyterlab_server/pull/192

Reply:
[Minor comment] It is a bad practice to set a mutable as default value.